electron/main.js

```diff
@@ -299,11 +310,11 @@ function showMainWindow() {
 
       if (isFullScreen) {
         main.once('leave-full-screen', () => {
-          main.hide();
+          closeButton();
         });
         main.setFullScreen(false);
       } else {
-        main.hide();
+        closeButton();
```

Comment:
@ConorIA Thanks for taking a look at this. We are not overly excited to add to the list of exceptions for specific environments but I guess there is currently no way around in order to address the issue. Can I ask you to name the function a little more verbose those? Possibly `onCloseMain` as the function is somewhat unrelated to its current name.